wordpress.org/public_html/wp-content/plugins/theme-directory/class-wporg-themes-upload.php

```diff
@@ -196,10 +198,10 @@ public function process_upload( $file_upload ) {
 		// We need a screen shot. People love screen shots.
 		if ( ! $this->has_screenshot( $theme_files ) ) {
 			/* translators: 1: screenshot.png, 2: screenshot.jpg */
-			return sprintf( __( 'The zip file must include a file named %1$s or %2$s.', 'wporg-themes' ),
+			array_push( $style_errors, sprintf( __( 'The zip file must include a file named %1$s or %2$s.', 'wporg-themes' ),
```

Comment:
I would personally avoid using `array_push()` and simply do `$style_errors[] = ...`
```suggestion
			$style_errors[] = sprintf( __( 'The zip file must include a file named %1$s or %2$s.', 'wporg-themes' ),
```

Reply:
Wasn't aware. I like it better for certain!